pkg/util/util.go

```diff
@@ -23,6 +23,7 @@ import (
 	"time"
 
 	"emperror.dev/errors"
+	samara "github.com/Shopify/sarama"
```

Comment:
There's a typo here
```suggestion
	"github.com/Shopify/sarama"
```
It `sarama`, not `samara`.

Reply:
Uhh. Nice catch. Will fix.